src/main/resources/templates/db_scripts/create_job_number_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS ${myuniversity}_${mymodule}.job_number (
+  last_number bigint NOT NULL DEFAULT 1,
+  type varchar(255) NOT NULL
```

Comment:
Better to add primary key here like this: PRIMARY KEY (type, last_number)
That will also create composite index, and in our UPDATE query 'type' is used at the first place so this composite index should help, because order of fields matter when creating indexes.

Also we can swap fields order in table for consistency, first field is type, second - last_number

Reply:
thanks, added